bluemira/radiation_transport/radiation_tools.py

```diff
@@ -365,31 +373,26 @@ def electron_density_and_temperature_sol_decay(
         radial decayed densities through the SoL. unit [1/m^3]
     """
     # temperature and density decay factors
-    t_factor = 7 / 2
-    n_factor = 1
+    if f_exp == 1:
+        t_factor = 7 / 2
+        n_factor = 1 / 3
+    else:
+        t_factor = 7
+        n_factor = 1 / 7
```

Comment:
where do these come from? Could we get a reference or some comment?

Reply:
@je-cook also parametrised these factors. Going to add references.